### DISCUSSION TITLE
make deploy error

### DISCUSSION THREAD

**Discussion by pwnzor11** [carlosedp/cluster-monitoring#179] [Q&A]
I get this error when using the make deploy command.  Everything worked up until that point

error: error reading [./manifests/setup/]: recognized file extensions are [.json .yaml .yml]

**Comment by pwnzor11** [comment]
https://github.com/carlosedp/cluster-monitoring/discussions/180

He wrote it up better.  Exact same problem

**Comment by carlosedp** [comment]
Fixed on a8c6bba.
